docs/notebooks/figanos_multiplots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>In this example the dots (scattermap) do not have the same cmap as the map, as can be illustrated by setting cmap='binary' in the fg.gridmap() call.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>